src/app-layout/__tests__/common.test.tsx

```diff
@@ -187,16 +187,6 @@ describeEachAppLayout(() => {
         expect(findToggle(wrapper).getElement()).toBe(document.activeElement);
       });
 
-      test('updates active element in controlled mode', () => {
```

Comment:
Is this test not relevant anymore?

Reply:
OK I see. Would it make sense to have coverage for the case of focusing via user interactions then? (Maybe by changing this test instead of removing it)

Reply:
this is covered in the integration tests